tests/integration/docusaurus/expectations/creating_custom_expectations/expect_parameterized_queried_column_value_frequency_to_meet_threshold.py

```diff
@@ -0,0 +1,118 @@
+"""
+This is a template for creating custom Parameterized TableQueryExpectations.
+For detailed instructions on how to use it, please see:
+    https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_table_query_expectations
+"""
+
+from typing import Dict, Optional, Union
+
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.execution_engine import ExecutionEngine
+from great_expectations.expectations.expectation import (
+    ExpectationValidationResult,
+    TableQueryExpectation,
+)
+
+
+class ExpectParameterizedQueriedColumnValueFrequencyToMeetThreshold(
+    TableQueryExpectation
+):
+    """Expect the frequency of occurrences of a specified <value> in a queried column to be at least <threshold> percent of values in that column."""
+
+    default_kwarg_values = {
+        "query": "SELECT col2, "
+        "CAST(COUNT(col2) AS float) / (SELECT COUNT(col2) FROM test) "
+        "FROM test "
+        "GROUP BY col2",
+        "value": "a",
+        "threshold": 0.4,
+    }
```

Comment:
love the way you're showing the baked-in query! I could also see the subclass masking that, as the default case.